src/Twilio/TwilioOrgsTokenAuth.cs

```diff
@@ -0,0 +1,160 @@
+using Twilio.Clients;
+using Twilio.Clients.BearerToken;
+using Twilio.Exceptions;
+using Twilio.Http.BearerToken;
+
+namespace Twilio
+{
+    /// <summary>
+    /// Default Twilio Client for bearer token authentication
+    /// </summary>
+    public class TwilioOrgsTokenAuthClient
+    {
```

Comment:
Name should be consistent 'TwilioOrgsTokenAuthClient' 
Either Bearer or Orgs.

Reply:
named bearer for all classes which is generic for token authentication. This class is named specific to orgs because this fetched the access token passing client id, client secret and grant type. 
The classes with name bearer in it can be re used with another client class which fetches token passing some other parameters?

Please suggest if this needs to be renamed to bearer?